### DISCUSSION TITLE
Can't find convert button

### DISCUSSION THREAD

**Discussion by nisargjoshi10** [hernanchavezthielemann/GRO2LAM#46] [Q&A] [answered]
Hi! I am using GRO2LAM for converting files. I installed the software by cloning the git repo and running the setup. Once, I have the software running I am able to load the files for conversion but the window does not show any `Convert` button for me to convert the files.

I tried installing the software with wget command too but seems like it still has the issue.

![image](https://user-images.githubusercontent.com/56214118/105745237-eb6fc780-5ef2-11eb-8d7f-ae3b9633687e.png)

I am on mac OS Mojave 10.14 and I'm creating a conda environment to run GRO2LAM.

Is there any recent update I should go with? Would really appreciate some help here.

Thanks,
Nisarg

**Comment by hernanchavezthielemann** [comment, marked as answer]
Hi Nisarg @nisargjoshi10 ,

To be honest I have never used/tested it on any mac OS, an that behavior that you describe is not something that happens on Ubuntu.

To solve what you suggest first I would like you to try resizing the window, like in the following image:
![resize_window](https://user-images.githubusercontent.com/29739618/105764842-4676de00-5f36-11eb-88e9-f20a3313317d.gif)

Let me know if that is possible, also if you can make it run the examples with your configuration ( I'm not sure if it could handle the folder separator properly in your system).

Best regards,
Hernan

**Comment by nisargjoshi10** [comment]
Hi Hernan,

That worked out perfectly! Thanks for the solution.

Best,
Nisarg

**Comment by hernanchavezthielemann** [comment]
Great, good to know!
so maybe, with conda, it could work under windows too!?

best,
Hernan

**Comment by nisargjoshi10** [comment]
Yeah it definitely should!

**Comment by hernanchavezthielemann** [comment]
The only drawback is that, for some reason, the menu bar is not in there, or is it after you expanded it? I don't think so
in any case, all that means is that you cannot go back, and the impossibility to access to the help/about button.

**Comment by nisargjoshi10** [comment]
No I couldn't find the menu bar even after expanding it. Also, you planning to upgrade the system from Python 2.7?

**Comment by hernanchavezthielemann** [comment]
I thought it would be like that ...
It is almost upgraded, I'm in the final tests.
Nevertheless, you can install the python3-beta version installing it with the beta link in the initial page Readme.
